src/item.h

```diff
@@ -67,6 +67,8 @@ struct damage_unit;
 class map;
 struct item_comp;
 
+using material_id_set = std::set<material_id>;
```

Comment:
Aliases for aggreagate types like this only obscure how the code works, forcing you to either guess what the type is from the name, or track down this using directive to figure it out.